addons/l10n_fr_invoice_addr/views/report_invoice.xml

```diff
@@ -0,0 +1,54 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <template id="report_invoice_document" inherit_id="sale.report_invoice_document_inherit_sale">
+        <xpath expr="//address" position="after">
+            <t t-set="partner" t-value="o.partner_id.parent_id or o.partner_id"/>
+            <t t-if="o.l10n_fr_is_company_french and partner.siret">
+                SIRET: <t t-esc="partner.siret"/>
+            </t>
+        </xpath>
+
+        <xpath expr="//address" position="attributes">
+            <attribute name="class">mb-0</attribute>
```

Comment:
changing the report for all companies, even the non french ones.

Can you show visually what style you modify with that?

Reply:
![Untitled](https://github.com/odoo/odoo/assets/163829053/c6ac0584-213a-4e23-a2d1-f814deff4865)

It removes the margin between the address and the SIRET. I agree that this should be conditional. I'll change that

Reply:
I did it albeit in a hacky way:
```xml
<xpath expr="//address" position="attributes">
    <attribute name="class">mb-0</attribute>
</xpath>

<xpath expr="//address" position="after">
     ... (add siret here) ...
     <div class="mb-3"/>
</xpath>
```

This will guarantee the padding gets added after the SIRET. My 1st idea was to use `t-attf-class` but I couldn't get it to execute in a Python context, only JS